src/components/learning-center/learning-searchbar.tsx

```diff
@@ -66,7 +67,7 @@ class LearningSearchBar extends Component<Props,State> {
     <div className="search-bar">
       <InstantSearch
         searchClient={algoliasearch(appId,searchKey)}
-        indexName="learning_center"
+        indexName={"learning_center" + (this.props.locale ? "_" + this.props.locale : "")} 
```

Comment:
In our actual search bar widget, all I needed to do was dynamically define the name of the search index based on the page's locale— thanks to us now having separate search indexes for each language. There was some other minor tooling to also add the locale prefix to the slugs that our hits link to.

Reply:
NOICE